packages/app/src/compositions/useAuthentication.ts

```diff
@@ -0,0 +1,23 @@
+import { ref, computed } from 'vue';
+import { User } from '@project-leek/commons';
+
+import feathers from '../lib/feathers';
+
+export const user = ref<User | null>(null);
+
+// !!user.value to cast the user variable to a boolean
+export const isAuthenticated = computed(() => !!user.value);
+
+export async function load() {
+  try {
+    await feathers.reAuthenticate();
+    user.value = (await feathers.get('authentication')).user;
```

Comment:
Do we need `await` here?

Reply:
It is needed. See [docs](https://docs.feathersjs.com/api/authentication/client.html#app-get-authentication)